corehq/apps/app_manager/models.py

```diff
@@ -987,6 +987,7 @@ def wrapped_xform(self):
     def validate_form(self):
         vc = self.validation_cache
         if vc is None:
+            # todo: now that we don't use formtranslate, does this still apply?
```

Comment:
OK fine, thanks for prodding me.

Looks like this was added in here for a specific reason, to fix a bug @emord was encountering. Commit here: https://github.com/dimagi/commcare-hq/commit/010461d3a7232239ffdc4952388fe883ce342d03, some discussion on the PR here: https://github.com/dimagi/commcare-hq/pull/10346#r61318829.

@emord do you think this is still necessary with formplayer?